packages/kubernetes-api-app/src/login/AuthLoginPage.tsx

```diff
@@ -1,27 +1,40 @@
-import React from 'react'
-import { usePlugins, DEFAULT_APP_NAME, useHawtconfig } from '@hawtio/react'
+import React, { useEffect } from 'react'
+import { usePlugins, DEFAULT_APP_NAME, useHawtconfig, useUser } from '@hawtio/react'
 import { backgroundImages, hawtioLogo } from './img'
 import { ListItem, ListVariant, LoginFooterItem, LoginPage } from '@patternfly/react-core'
 import { log } from '@hawtio/online-oauth'
 import { AuthLoadingPage } from './AuthLoadingPage'
+import { useNavigate } from 'react-router-dom'
 
 export const AuthLoginPage: React.FunctionComponent = () => {
+  const navigate = useNavigate()
+
+  const { isLogin, userLoaded } = useUser()
   const { hawtconfig, hawtconfigLoaded } = useHawtconfig()
   const { plugins, pluginsLoaded } = usePlugins()
 
-  if (!hawtconfigLoaded && !pluginsLoaded) {
+  useEffect(() => {
+    if (isLogin) {
+      navigate('/')
+    }
+  }, [isLogin, navigate])
```

Comment:
Looks like you separated the pull reqs but haven't looked at my previous reviews at all?

I said:
While I know hawtio/react does the same code, now I wonder why we need `useEffect`. (Sorry it must be me who first introduced it...)  Can't we just put it in the main body of the component? :thinking:

Reply:
I did look at your comments. Although, not got to all of them. Rather than change the `useEffect` here, I was going to look at it in hawtio/next first and then consider changing them here after we had come to a conclusion. However, I can do it the opposite way around if preferred.

Reply:
To address the comment. Testing each of the test-apps with the `useEffect` removed leaves the page stuck on the `Loading...` screen. Despite logging-in the page never refreshes.